lib/countly.js

```diff
@@ -2444,14 +2444,14 @@
         switch(key) {
         //queue of requests
         case Countly.namespace + "cly_queue":
-            requestQueue = Countly.deserialize(e.newValue);
+            requestQueue = Countly.deserialize(e.newValue || []);
             break;
             //queue of events
         case Countly.namespace + "cly_event":
-            eventQueue = Countly.deserialize(e.newValue);
+            eventQueue = Countly.deserialize(e.newValue || []);
             break;
         case Countly.namespace + "cly_remote_configs":
-            remoteConfigs = Countly.deserialize(e.newValue);
+            remoteConfigs = Countly.deserialize(e.newValue || {});
```

Comment:
`deserialize` must receive a string to JSON.parse, right? Not actual objects or arrays, I think.

Reply:
See: https://github.com/Countly/countly-sdk-web/blob/master/lib/countly.js#L2547-L2553

Reply:
thank you @mt-ronkorving ! Updated.

Anything else that you think needs to be added here? Anything I need to do get the minified JS updated? cc @ar2rsawseen 

Thanks again team!